socorro/external/crashstorage_base.py

```diff
@@ -97,50 +97,6 @@ def as_memory_dumps_mapping(self):
         return in_memory_dumps
 
 
-class Redactor(RequiredConfig):
```

Comment:
We're no longer redacting, so we can drop this class which isn't really doing the right thing anymore anyhow.